### DISCUSSION TITLE
NextUI Select Component is not Opening when inside ShadCN UI Sheet / Dialog

### DISCUSSION THREAD

**Discussion by zaidsoma** [heroui-inc/heroui#3553] [Help]
I have been trying to use Nextui together with Shadcn but when i try to place a Select Element From NextUi inside Shadcn's sheet component which opens a dialog it just won't open. 

any help and guidance is appreciated

**Comment by zaidsoma** [comment]
It seems to be working when i add modal={false} to sheet component is there another work around?

**Reply by monterourena** [reply]
Hi @zaidsoma thank you very much! There are some drawbacks (e.g. the backdrop disappears) but at least for me it's not a problem. Thanks, probably later with a little more time I'll try to experiment with the Portals stuff and see if I find another solution.

**Reply by Zegoverno** [reply]
When modal is set to false, if a move the mouse it closes the dialog? Anyone got this problem as well?
